fe/fe-core/src/main/java/org/apache/doris/qe/StmtExecutor.java

```diff
@@ -646,6 +646,9 @@ public void executeByLegacy(TUniqueId queryId) throws Exception {
         profile.getSummaryProfile().setQueryBeginTime();
         context.setStmtId(STMT_ID_GENERATOR.incrementAndGet());
         context.setQueryId(queryId);
+        if (parsedStmt == null) {
+            parseByLegacy();
+        }
```

Comment:
why add parsedStmt == null? we already check parsedStmt == null in parseByLegacy()